Sources/StreamVideo/CallSettings/SpeakerManager.swift

```diff
@@ -0,0 +1,77 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// Handles the speaker state during a call.
+public class SpeakerManager: ObservableObject, CallSettingsManager {
+        
+    internal let callController: CallController
+    @Published public internal(set) var status: CallSettingsStatus
+    @Published public internal(set) var audioOutputStatus: CallSettingsStatus
+    internal let state = CallSettingsState()
+    
+    init(
+        callController: CallController,
+        initialStatus: CallSettingsStatus,
```

Comment:
`initialStatus` and `audioOutputStatus` are not clear to me what to me. What about `initialSpeakerStatus` and initial `initialAudioOutputStatus`?

Reply:
changed